src/stratis_cli/_error_reporting.py

```diff
@@ -187,19 +187,16 @@ def _interpret_errors(errors):
         ):
             return "Most likely stratis is unable to connect to the stratisd D-Bus service."
 
-        # pylint: disable=fixme
-        # FIXME: remove no coverage pragma when adequate testing for CLI output
-        # exists.
         if (
             # pylint: disable=bad-continuation
             isinstance(error, dbus.exceptions.DBusException)
             and error.get_dbus_name() == "org.freedesktop.DBus.Error.NoReply"
-        ):  # pragma: no cover
+        ):
```

Comment:
Rather than assigning it to ```fmt_str``` you should probably just return it directly in the same way as on line 188.